From 393e5bc2ec26957d70228ac4a4b99455c18110d5 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Thu, 20 Aug 2015 21:45:58 +0800
Subject: [PATCH 0247/5242] MLK-11389-02 ARM: dts: imx: add M/F mix property
 for imx6sx

commit  903885c76cfca08953a6760f904258b4c4758567 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 'fsl,mf-mix-wakeup-irq' property used for M/F mix in gpc node
on i.MX6SX.

Signed-off-by: Bai Ping <b51503@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 1a4e27e..7774228 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -776,6 +776,7 @@
 				#interrupt-cells = <3>;
 				interrupts = <GIC_SPI 89 IRQ_TYPE_LEVEL_HIGH>;
 				interrupt-parent = <&intc>;
+				fsl,mf-mix-wakeup-irq = <0x7c00000 0x3d00 0x0 0x400240>;
 				clocks = <&clks IMX6SX_CLK_IPG>;
 				clock-names = "ipg";
 
-- 
1.7.9.5

